crm/main.py

```diff
@@ -211,15 +211,20 @@ def cli(ctx):
     # Use the db object if given as input.
     db = None if ctx.obj is None else ctx.obj.cfgdb
 
+    # Note: SonicDBConfig may be already initialized in unit test, then skip
+    if not SonicDBConfig.isInit():
```

Comment:
isInit() is the check for SonicDBConfig.initialize(), good if it could be moved below in else.